### PR TITLE
add(epp): emc ppma rest api

### PR DESCRIPTION
**Pull request by garnier-quentin**
## Target serie

- [X] 20.04.x
- [X] 20.10.x
- [X] 21.04.x (master)

### CODE REVIEW

File: en/integrations/plugin-packs/procedures/applications-emc-ppma-restapi.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+title: EMC PPMA Rest API
+---
+
+## Plugin-Pack Assets
+
+### Monitored Objects
+
+The Plugin-Pack EMC PowerPath Management Appliance collects metrics for:
+* Hosts
+
```
Comment (Sims24) [inline en/integrations/plugin-packs/procedures/applications-emc-ppma-restapi.md:12]: - Missing host discovery part. Check: https://github.com/centreon/centreon-documentation/blob/master/en/integrations/plugin-packs/procedures/cloud-gcp-storage.md#discovery-rules

File: en/integrations/plugin-packs/procedures/applications-emc-ppma-restapi.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+| Metric name                            | Description                                                  | Unit |
+| :------------------------------------- | :----------------------------------------------------------- | :--- |
+| host status                            | Host powerpath status, possible to set string-based alerts   |      |
+| *hostname*#host.paths.total.count      | Number of total paths for the host                           |      |
+| *hostname*#host.paths.dead.count       | Number of dead paths for the host                            |      |
+| *hostname*#host.volumes.total.count    | Number of volumes attached                                   |      |
+| *hostname*#host.volumes.dead.count     | Number of dead volumes attached                              |      |
+| *hostname*#host.volumes.degraded.count | Number of degraded volumes attached                          |      |
+
+
```
Comment (Sims24) [inline en/integrations/plugin-packs/procedures/applications-emc-ppma-restapi.md:28]: extra line break

File: en/integrations/plugin-packs/procedures/applications-emc-ppma-restapi.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+| :-------- | :------------------ | :------------------------------------------------------------------------- |
+| X         | PPMAAPIPORT         | Port used (Default: 443)                                                   |
+| X         | PPMAAPIPROTO        | Specify https if needed (Default: 'https')                                 |
+| X         | PPMAAPIUSERNAME     | EMC PPMA username                                                          |
+| X         | PPMAAPIPASSWORD     | EMC PPMA password                                                          |
+|           | PPMAAPIEXTRAOPTIONS | Any extra option you may want to add to the command (eg. a --verbose flag) |
+
+By default, Host Template *App-Emc-Ppma-Restapi* doesn't have Service Template attached. You could:
+ * attached Service Template to Host Template *App-Emc-Ppma-Restapi-custom*
+ * using service discovery
```
Comment (Sims24) [inline en/integrations/plugin-packs/procedures/applications-emc-ppma-restapi.md:85]: s/You could:/You may want to:/
s/attached/attach/
s/using/use/

File: en/integrations/plugin-packs/procedures/applications-emc-ppma-restapi.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+
+## FAQ
+
+### How to check in the CLI that the configuration is OK and what are the main options for ?
+
+Once the Plugin installed, log into your poller using the *centreon-engine* user account and test by running the following command
```
Comment (Sims24) [inline en/integrations/plugin-packs/procedures/applications-emc-ppma-restapi.md:91]: uppercase Poller

File: fr/integrations/plugin-packs/procedures/applications-emc-ppma-restapi.md [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+title: EMC PPMA Rest API
+---
+
+## Contenu du Plugin-Pack
+
+### Objets supervisés
+
+Le Plugin-Pack EMC PowerPath Management Appliance collecte les données pour:
+* Hosts
+
```
Comment (Sims24) [inline fr/integrations/plugin-packs/procedures/applications-emc-ppma-restapi.md:12]: See EN comment about the discovery rules part.

**Review by Sims24 — CHANGES_REQUESTED** [review]

File: en/integrations/plugin-packs/procedures/applications-emc-ppma-restapi.md (hunk 1 above)
Comment (garnier-quentin) [reply en/integrations/plugin-packs/procedures/applications-emc-ppma-restapi.md:12]: there is no host discovery.

Comment (Sims24) [reply en/integrations/plugin-packs/procedures/applications-emc-ppma-restapi.md:12]: we will add it later

**Review by Sims24 — APPROVED** [review]

**Pull request merged**